src/main/java/com/traveltime/sdk/dto/responses/timemapfast/ResponseProperties.java

```diff
@@ -0,0 +1,16 @@
+package com.traveltime.sdk.dto.responses.timemapfast;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ResponseProperties {
```

Comment:
should it be empty?

Reply:
This endpoint never returns any properties but I need a response type for geojson FeatureCollection<G, P> perhaps there is a better way of doing this is you could advise? If I would reuse existing one from current time-map we would jsut be adding needless nulls to response.

Reply:
got it, so leave it like that